### ISSUE TITLE
Do not occupy default django tcp port when starting pgmanage-server process

### ISSUE THREAD

**Issue by ccurvey** [commandprompt/pgmanage#135]
I do a lot of Django development -- it would be nice if there was a way to tell pgManage to use a port other than 8000, as that conflicts with the Django development server.  To wit

1. Start pgManage
2. try to start your Django site with `python manage.py runserver`

```
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
Error: That port is already in use.
```

I know I can start the Django server with `python manage.py runserver localhost:8080` and use that, but it would be nice to be able to use it as-is

**Comment by eug3nix** [comment]
Hi @ccurvey 
I think that we could randomize the TCP port number when starting pgmanage back-end process for desktop variant of the app.
Thank for the suggestion!

**Comment by eug3nix** [comment]
fixed in [0223839](https://github.com/commandprompt/pgmanage/commit/0223839e85419c4783dd754e128c60e25ff5ed5d)
the fix will be included in the next release
